server/sms/nexmo_sms_sender.go

```diff
@@ -0,0 +1,193 @@
+package gateway
+
+import (
+	"bytes"
+	"encoding/json"
+	log "github.com/Sirupsen/logrus"
+	"github.com/smancke/guble/protocol"
+	"github.com/wendal/errors"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+)
+
+const (
+	KEY    = "ce40b46d"
+	SECRET = "153d2b2c72985370"
+	URL    = "https://rest.nexmo.com/sms/json?"
+)
+
+type NexmoSms struct {
```

Comment:
Keep type definition along side methods. You can even move this type in own file.